index.html

```diff
@@ -1,26 +1,81 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
 	<head>
-    	<meta charset="utf-8">
-    	<meta name="description" content="Your description goes here">
-    	<meta name="keywords" content="one, two, three">
+		<title>Relaxr</title>
+		<link rel="stylesheet" type="text/css" href="css/normalize.css"
+		
+		<link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,600,800" rel="stylesheet">
+		<link rel="stylesheet" type="text/css" href="css/style.css">
+	</head>
 
-		<title>Blank Template</title>
+	<body>
+		<header>
+			<div class="container">
+				<h1>Relaxr</h1>
+				<nav>
+					<a href="#">About</a>
+					<a href="#">FAQ</a>
+					<a href="#">Team</a>
+					<a href="#">Contact Us</a>
+					<a href="#">Blog</a>
+				</nav>
+			</div>
+		</header>
+		
+		<main class="clearfix">
+			<div class="article">
+				<article>
```

Comment:
Good choice of `<article>` 👍 . My only recommendation would be to wrap each blog post in its own `<article>` rather than wrapped both of them in a single one